src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorOverloadsHaveNamedAlternates.cs

```diff
@@ -1,69 +1,263 @@
         internal static DiagnosticDescriptor MultipleRule = new DiagnosticDescriptor(RuleId,
                                                                              s_localizableTitle,
                                                                              s_localizableMessageMultiple,
                                                                              DiagnosticCategory.Usage,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: MsdnUrl,
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
         internal static DiagnosticDescriptor VisibilityRule = new DiagnosticDescriptor(RuleId,
                                                                              s_localizableTitle,
                                                                              s_localizableMessageVisibility,
                                                                              DiagnosticCategory.Usage,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: MsdnUrl,
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(DefaultRule, PropertyRule, MultipleRule, VisibilityRule);
 
         public override void Initialize(AnalysisContext analysisContext)
-        { 
-            
+        {
+            analysisContext.RegisterSymbolAction(DoAnalysis, SymbolKind.Method);
+        }
+
+        private static void DoAnalysis(SymbolAnalysisContext symbolContext)
+        {
+            var methodSymbol = (IMethodSymbol)symbolContext.Symbol;
+            var location = methodSymbol.OriginalDefinition.Locations.First();
```

Comment:
You don't need to calculate the location unless you're reporting certain of the diagnostics below. I'd wrap this in a helper method `LocationOfSymbol`, and then call `LocationOfSymbol(methodSymbol)` in each of the three places below where you report a diagnostic at `location`.